spec/karma.conf.js

```diff
@@ -90,7 +90,7 @@ module.exports = function (config) {
 		// - Opera
 		// - Safari (only Mac)
```

Comment:
This comment should probably be updated.
```suggestion
		// - SafariNative (only Mac)
```